lib/mailjex/utils/comms.ex

```diff
@@ -32,9 +33,9 @@ defmodule Mailjex.Utils.Comms do
     end
   end
 
-  def request(:put, path, body) do
+  def request(:put, path, body, public_key, private_key) do
```

Comment:
The PUT version of `request` need some default values so we don't break the current api.

```suggestion
  def request(:put, path, body, public_key \\ nil, private_key \\ nil) do
```

Reply:
Ah checked it now, the default values are defined here https://github.com/PiggyPot/mailjex/pull/5/files/c6b109416ce88b7096fc67b95b14d7a4d95856ec#diff-990bfd45dfb16afa2fb76b46d9b25fbaR15 and applies to all definitions of request/5.